Makefile.common

```diff
@@ -253,17 +246,3 @@ $(GOLANGCI_LINT):
 		| sh -s -- -b $(FIRST_GOPATH)/bin $(GOLANGCI_LINT_VERSION)
 endif
 
-.PHONY: precheck
-precheck::
```

Comment:
This can be overriden in other projects

Reply:
See the link in the description: there isn't a place in either prometheus or prometheus-community where that is done?